### PR TITLE
chore: upgrade deps, adjust for linting rules

### PR DESCRIPTION
**Pull request by nomego**
Signed-off-by: Patrik Kullman <patrik.kullman@neovici.se>

### CODE REVIEW

File: cosmoz-templatizing-mixin.js [hunk 1]
```diff
@@ -234,7 +237,9 @@ export const templatizing = dedupingMixin(base => class extends base { // eslint
 		if (!instance) {
 			return;
 		}
-		const {children, root} = instance;
+		const {
+			children, root
```
Comment (JockeCK) [inline cosmoz-templatizing-mixin.js]: more of an general question. Should we always make stuff like this on two rows or is this ok?

Comment (cristinecula) [reply cosmoz-templatizing-mixin.js]: If it were up to me, I'd just throw standard prettier at the whole codebase and be done with it...

Comment (cristinecula) [reply cosmoz-templatizing-mixin.js]: and keep eslint only for semantic rules (undefined vars, unused vars, etc)

Comment (nomego) [reply cosmoz-templatizing-mixin.js]: Well it's a tricky rule. For objects, the rule now is that properties should be written on their own lines if there are two or more, but the same enforcement doesn't work for destructuring and imports even though they look similar. But I think it makes sense to write out object syntax over multiple lines, the diffs get cleaner and from most aspects it should be easier to read as well.

**Comment by cristinecula** [comment]
@nomego I think you should rewrite the commit messages, so they are all `chore` or `build`, otherwise it will trigger a version bump, when it is unnecessary

**Comment by nomego** [comment]
Ok I'll just squash it.

**Pull request merged**